db/migrate/20181210145503_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :title
```

Comment:
Don't forget to add the null constraint so that the model and the db are consistent.